docs/runtime/compiler-ir.md

```diff
@@ -9,15 +9,74 @@ multiple passes. Every pass is a class implementing the
 See [Runtime roadmap - static analysis](../runtime-roadmap.md#static-analysis)
 for future goals.
 
-## Visualization
-
-The IR can be visualized using `--vm.D=enso.compiler.dumpIr` system property.
-This will output a `.dot` file in [GraphViz](www.graphviz.org) format in the
-`ir-dumps` directory for each IR in the program. The _dot_ file format is a
-minimal textual format, that can be converted to a graphical representation
-using the `dot` command from the GraphViz package. For example, on Ubuntu,
-install `dot` with `sudo apt install graphviz`. Then, convert the `.dot` file to
-a `.svg` image with `dot -Tsvg -o <output>.svg <input>.dot`. An example is:
-![image.svg](https://github.com/user-attachments/assets/26ab8415-72cf-46da-bc63-f475e9fa628e)
-
-See `org.enso.compiler.dump.IRDumper`.
+## Dumping IR
+
+The IR can be visualized using the `enso.compiler.dumpIr` system property. The
```

Comment:
Documentation for [working with IGV](https://github.com/enso-org/enso/blob/2e714a70ddf12456e9f3fa9e132fd2ac43aa3b77/tools/enso4igv/IGV.md#using-the-igv) including how to dump graphs already exists. These two documentations should be connected/merged together.